Makefile

```diff
@@ -51,12 +54,17 @@ GO_TEST_OPTS                 += -mod=readonly -failfast -race
 GO_OUT_BIN_DIR               := ./dist
 
 ## General Variables
+# Branch Variables
+PROTECTED_BRANCH := main
```

Comment:
Should this be `master` or are we planning to switch? I am fine either way, in fact the plugin's repo defaults to `main`.

Reply:
Nice catch. I was under the impression that the protected branch was `main` for the rtcd project. Will change that to main 👍

Reply:
Ideally we should eventually move everywhere to the new naming conventions